exercises/practice/yacht/yacht.h

```diff
@@ -0,0 +1,27 @@
+#ifndef YACHT_H
+#define YACHT_H
+
+#include <stdint.h>
+
+typedef enum {
+   ONES,
+   TWOS,
+   THREES,
+   FOURS,
+   FIVES,
+   SIXES,
+   FULL_HOUSE,
+   FOUR_OF_A_KIND,
+   LITTLE_STRAIGHT,
+   BIG_STRAIGHT,
+   CHOICE,
+   YACHT
+} category_t;
+
+typedef struct {
+   uint8_t faces[5];
+} dice_t;
+
+int score(dice_t dice, category_t category);
```

Comment:
In the structs I care about the size (I want them to be as small as possible). Where I use `int`, I'll gladly accept whatever's the platform's default (it will be at least `int16_t`, so good enough). I do find that consistent. But I'm following @ryanplusplus's argument that this is too surprising for students. I changed it to `int` now. Although I do think that there should not be a hard distinction between "production" and "practice" code. Code quality matters, always. However, this is a micro-optimization that even in production code could lead to some debates. It would depend on the context if the performance benefit warranted the increased potential for confusion. Here, they probably do not.